firebase/functions/package.json

```diff
@@ -9,7 +9,7 @@
     "node_modules": "rm -rf ./node_modules && npx yarn install",
     "build:pre": "npm run node_modules && npm run clean_dist && npm run mk_dist && npm run copy",
     "build": "npm run build:pre && tsc",
-    "deploy": "npm run build && firebase deploy && npm run clean_dist",
+    "deploy": "npm run build && npx firebase deploy && npm run clean_dist",
```

Comment:
This should automatically pick the one from `node_modules`. So it should also work without `npx`.